src/components/matchHistory/MatchDetails.tsx

```diff
@@ -26,9 +26,16 @@ export const MatchDetails = React.memo(function MatchDetails() {
 
     const title = `GAME ${match.id}`;
 
+    const rankDictionary: { [key: string]: string } = {
```

Comment:
move to to outside of component-- we don't want to regenerate this dictionary on every render, it should never change